sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Acl.java

```diff
@@ -42,14 +42,33 @@
 
 
   /**
-   * The gRPC status code to respond with when access is denied.
+   * The status code to respond with when access is denied.
    *
-   * By default, this will be 7 (permission denied), but alternatives might include 16 (unauthenticated) or 5 (not
-   * found). If 0, indicates that the code should be inherited from the parent (regardless of the inherit field).
+   * By default, this will be 'Forbidden', but alternatives might include 'Authentication required' or 'Not
+   * Found'. If 'Inherited', indicates that the code should be inherited from the parent (regardless of the inherit field).
    *
-   * When HTTP transcoding is in use, this code will be translated to an equivalent HTTP status code.
    */
-  int denyCode() default 0;
+  DenyStatusCode denyCode() default DenyStatusCode.FORBIDDEN;
```

Comment:
So, here instead of `FORBIDEN`, we should not have a default. The default should be `null` and then we translate to `0`.

Reply:
I realise not that there is some confusion here. We say that the default is `7` (permission denied), but then the default was `0`. 

That was wrong, the default should be indeed `DenyStatusCode.FORBIDDEN`, mapped to `7`